src/app/modules/customer-management/store/customer-management.effects.ts

```diff
@@ -93,8 +93,9 @@ export class CustomerEffects {
   unarchiveCustomer$: Observable<Action> = this.actions$.pipe(
     ofType(actions.CustomerManagementActionTypes.UNARCHIVE_CUSTOMER),
     map((action: actions.UnarchiveCustomer) => ({
+
       id: action.payload,
-      status: 'active',
+      status: action.status,
```

Comment:
did you check if it is updating correctly in back

Reply:
yes, we checked, we found no problems